### PR TITLE
new feature remote controll

### PR DESCRIPTION
**Pull request by KRav007**
add new participants with a Json string to the CPD

### CODE REVIEW

**Comment by c-schmitz** [comment]
Why JSON? I don't see any advantage in this. The same appoach as add_participants should be used. There is also no documentation on the last param '$sAction' - what is it for?

**Comment by KRav007** [comment]
Thought JSON could be nice, if you like to connect to JavaScript applications like angular or using ajax.

$sAction - forgot the docu. It's more something for the future if you like to update a participant from outside. But if I think about it, you can delete it. And if a 2nd version of the function comes, we can use $sAction = null, in order not irritate some systems that use it meanwhile.

**Comment by SamMousa** [comment]
The remote control uses JSON / XMLRPC anyway, so it makes no sense to encode parameters..

This would either result double json encoding being sent over the network, or json inside xml...

**Comment by KRav007** [comment]
ok makes sense. I change to array.

**Comment by KRav007** [comment]
Changed Json to array

**Comment by KRav007** [comment]
removed duplicate content
